build.py

```diff
@@ -56,7 +56,7 @@
     "i386": None,
     "x86": None,
     "x86_64": None,
-    "arc": "arc-linux-",
+    "arc": "arc-elf32-",
```

Comment:
`build.py` is not used any more.  See `kernelci/build.py` instead.

Reply:
Ah the other change in `build-configs.yaml` should be used in `kernelci/build.py` so that should be fine.